plugins/outputs/opensearch/README.md

```diff
@@ -0,0 +1,373 @@
+# Opensearch Output Plugin
+
+This plugin writes to [Opensearch](https://opensearch.org/) via HTTP using
+[Elastic](http://olivere.github.io/elastic/) client API.
+
+It supports Opensearch releases from 1.x up to 2.x.
```

Comment:
Was there a 0.x release? If not we can just say:

```suggestion
It supports OpenSearch releases 1 and 2. Future comparability with 1.x is not
guaranteed and instead will focus on 2.x support. Consider using the existing
Elasticsearch plugin for 1.x.
```

Reply:
done